examples/ng_networking/Makefile

```diff
@@ -0,0 +1,35 @@
+# name of your application
+APPLICATION = ng_networking
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= native
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../..
+
+# Include packages that pull up and auto-init the link layer.
+# NOTE: 6LoWPAN will be included if IEEE802.15.4 devices are present
+USEMODULE += ng_netif_default
+USEMODULE += auto_init_ng_netif
```

Comment:
Is this still needed? Can't this be included automatically if `auto_init` and `ng_netif` are included?